packages/gatsby-source-contentful/package.json

```diff
@@ -18,6 +18,7 @@
     "contentful": "^7.14.12",
     "fs-extra": "^9.0.1",
     "gatsby-core-utils": "^1.5.0-next.0",
+    "gatsby-plugin-image": "^0.2.0-next.0",
```

Comment:
We'll want this to be a peer-dependency and probably on require it when gatsbyImageData is used.

**Why?**
Npm doesn't dedupe packages so we might end up with multiple gatsby-plugin-images on a different version. Which could end in different results.

Reply:
The issue only requireing it when it's used is that it uses the helpers in the package to set up the resolver. We'd not be able to define the node type unless it's available

Reply:
Maybe use schema customization? https://www.gatsbyjs.com/docs/schema-customization/